applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -474,18 +476,22 @@ int le_audio_send(uint8_t const *const data, size_t size)
 	//TODO: Handling dual channel sending properly
 	net_buf_add_mem(buf, data, 120);
 
-#if (CONFIG_AUDIO_SOURCE_I2S)
 	struct bt_iso_tx_info tx_info = { 0 };
 
 	ret = bt_iso_chan_get_tx_sync(audio_stream.iso, &tx_info);
 
 	if (ret) {
 		LOG_WRN("Error getting ISO TX anchor point: %d", ret);
-	} else {
+	}
+
+#if (CONFIG_AUDIO_SOURCE_I2S)
+	if (tx.info.ts != 0) {
```

Comment:
1) tx.info = tx_info right?

2) Isnt it better to only use tx_info if bt_iso_chan_get_tx_sync() returns no error?

Reply:
Nice catch, added `&& !ret` to the if-statement